scriptlets/install_checkbox_snaps.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Installs Checkbox and exports the installed version on the DUT
```

Comment:
I would slightly amend that to "Installs the Checkbox snap on the device/DUT and exports the installed version to...".